pkg/models/app.go

```diff
@@ -12,3 +12,10 @@ type App struct {
 	DeployedVersions      *[]Version `json:"deployedVersions,omitempty"`
 	DeletedAt             string     `json:"deletedAt,omitempty"`
 }
+
+func NewAppByNameAppID(name, appId string) *App {
```

Comment:
This function is not being used anywhere, why is it being added?

Reply:
It will be used by the operator.